agenda-webapps/src/main/webapp/vue-app/agenda/components/AgendaToolbar.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
```

Comment:
No need to have a div as parent of the Vue component, since you have one Vue sub component.